packages/xo-web/src/common/intl/messages.js

```diff
@@ -804,6 +804,10 @@ const messages = {
   srShared: 'Shared',
   srNotShared: 'Not shared',
   pbdNoSr: 'No storage detected',
+  deletePbdsModalTitle: 'Delete PBD{nPbds, plural, one {} other {s}}',
+  deletePbdsModalMessage:
+    'Are you sure you want to delete {nPbds, number} PBD{nPbds, plural, one {} other {s}}?',
```

Comment:
@olivierlambert When the user clicks "Forget" in SR > Hosts, should the modal then say "Delete PBDs" as it's actually what it does or should it say something more user friendly like "Forget hosts"?

Reply:
We could find a middle ground, with a short explanation of what's a PBD. But I don't want hide all the complexity. Eg: "This will disconnect the SR to all the host by removing all their link to it (PBDs)"